src/compiler/parser.ts

```diff
@@ -6441,7 +6441,6 @@ namespace ts {
                                     // for malformed examples like `/** @param {string} x @returns {number} the length */`
                                     state = JSDocState.BeginningOfLine;
                                     margin = undefined;
-                                    indent++;

```

Comment:
this was just wrong, as far as I can tell, but only was used for tags followed by a newline:

```js
@ex
stuff
```

Reply:
That is, all our previous tests cases looked like this:

```js
@param x   comment
           comment
              indent
```

Never like this:

```js
@example
comment
comment
  indent
```

so this erroneous increment was hidden by the fact that `indent` had already passed `margin`.